config.toml

```diff
@@ -1,14 +1,14 @@
 [model]
-hidden_sizes = [3000, 3000, 3000]
```

Comment:
The old file is technically a little better, but I like the smooth loss for latent softmax classifier. So I am making the change. It is only a little worse.